src/api/Controllers/FaciesDescriptionController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using BDMS.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace BDMS.Controllers;
+
+[ApiController]
+[Route("api/v{version:apiVersion}/[controller]")]
+public class FaciesDescriptionController : BdmsControllerBase<FaciesDescription>
+{
+    private readonly BdmsContext context;
+
+    public FaciesDescriptionController(BdmsContext context, ILogger<FaciesDescription> logger)
+        : base(context, logger)
+    {
+        this.context = context;
+    }
+
+    /// <summary>
+    /// Asynchronously gets the <see cref="FaciesDescription"/>s, optionally filtered by <paramref name="stratigraphyId"/>.
+    /// </summary>
+    /// <param name="stratigraphyId">The id of the stratigraphy referenced in the facies descriptions to get.</param>
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<FaciesDescription>>> GetAsync([FromQuery] int? stratigraphyId = null)
```

Comment:
PP: Da wir in dieser Methode immer nur Rückgabewerte vom Typ `IEnumerable<FaciesDescription>` zurückgeben, kann hier direkt `Task<IEnumerable<FaciesDescription>>` verwendet werden und auf `ActionResult` verzichtet werden.